src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1552,7 +1555,8 @@ public void run() {
             r.getRepo().install(serviceType);
           }
         } catch (Exception e) {
-          r.error(e);
+          r.error("dependencies failed - install error", e);
+          throw new RuntimeException(String.format("dependencies failed - install error %s", e.getMessage()));

```

Comment:
Does this cause the install to exit if it fails?  (I believe that's a good behavior. just curious)

Reply:
not sure, ivy in general doesn't throw, but the install process does log out errors to a preserved myrobotlab-install.log
its been slightly improved upstream, but still room for clear notification for the user